modules/cef/cef-plugin.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2015 Balabit
+ * Copyright (c) 2015 bkil.hu
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ */
+
+#include "format-cef-extension.h"
+#include "plugin.h"
+#include "plugin-types.h"
+
+static Plugin cef_plugins[] =
+{
+  TEMPLATE_FUNCTION_PLUGIN(tf_cef, "format-cef-extension"),
+};
+
+gboolean
+cef_plugin_module_init(GlobalConfig *cfg, CfgArgs *args)
+{
+  plugin_register(cfg, cef_plugins, G_N_ELEMENTS(cef_plugins));
+  return TRUE;
+}
+
+const ModuleInfo module_info =
+{
+  .canonical_name = "cef-plugin",
```

Comment:
So should we change these in JSON as well?
modules/json/json-plugin.c

On Mon, Dec 21, 2015 at 8:49 AM, Tibor Benke <notifications@github.com>
wrote:

> In modules/cef/cef-plugin.c
> <https://github.com/balabit/syslog-ng/pull/842#discussion_r48122978>:
>
> > +
> > +static Plugin cef_plugins[] =
> > +{
> > +  TEMPLATE_FUNCTION_PLUGIN(tf_cef, "format-cef-extension"),
> > +};
> > +
> > +gboolean
> > +cef_plugin_module_init(GlobalConfig *cfg, CfgArgs *args)
> > +{
> > +  plugin_register(cfg, cef_plugins, G_N_ELEMENTS(cef_plugins));
> > +  return TRUE;
> > +}
> > +
> > +const ModuleInfo module_info =
> > +{
> > +  .canonical_name = "cef-plugin",
>
> Just a nitpick, but I'd use just cef here (as its directory name). Also,
> I'd rename cef_plugin_module_init to cef_module_init.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/842/files#r48122978>.
>